lib/eyaml/railtie.rb

```diff
@@ -9,7 +9,13 @@ class Railtie < Rails::Railtie
       PRIVATE_KEY_ENV_VAR = "EJSON_PRIVATE_KEY"
 
       config.before_configuration do
-        secrets_files.each do |file|
+        secrets_or_credentials = if Rails.version.start_with?("7.2") || Dir.glob(Rails.root.join("config", "credentials.*")).any?
```

Comment:
What happens if both files are present? As in - do we need to force the user to have both, or to choose one or the other? can we conditionally assign both, and let `credentials` take over (with a deep merge, for example)? Or we could tell the user that they need to rename the file and it will get picked up automatically? because there are Rails versions which have both (like we have now)

Reply:
Also: this check is too magicall as well, as it would pick up any, say, `config.bak.rb` file as an indication that the application is using credentials - while it might not be the case.

Reply:
I have added a more explicit check for both kinds. From rails 7.2 when secret files are deprecated we only look for credential files here. Before that credential files have prio over secrets, but secrets will still work of course.